coding-exercise-kotlin/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>coding-exercise</artifactId>
+        <groupId>com.magicline</groupId>
```

Comment:
`com.spa`?

Reply:
didnt want to mess around with the already existing parent-pom group identifiers. can do in a separate PR

Reply:
all fine just nit